index.js

```diff
@@ -156,9 +161,15 @@ function processContains(/* CODE HERE */) {
  * [2] Invoking `processDuplicateFree` passing `[1,1,2,2,3]` and `(arr) => arr.length`,
  * should return 3.
 */
-function processDuplicateFree(/* CODE HERE ONLY AFTER COMPLETING ALL OTHER TASKS */) {
-  /* CODE HERE ONLY AFTER COMPLETING ALL OTHER TASKS */
-}
+function processDuplicateFree(list, callback) {
+  let noDuplicates = [];
+  for(let i = 0; i < list.length; i++) {
+    if(noDuplicates.includes(list[i]) != true){
+      noDuplicates.push(list[i]);
+    };
+  };
+  return callback(noDuplicates);
+};
 
```

Comment:
This is a really great and logical solution.  Another option would be to use the `.filter` method and then use the `.includes` in the callback function that is passed to the filter method.  With the filter method it automatically returns a new array so you also wont have to manually create one.